ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -0,0 +1,376 @@
+    id: root
+
+    property alias currentTabIndex: stackLayout.currentIndex
+    property bool isCurrentUser
+    property bool readOnly
+    property var profileStore
+    property var walletStore
+    property var communitiesModel
+
+    signal closeRequested()
+
+    horizontalPadding: readOnly ? 20 : 40 // smaller in settings/preview
+    topPadding: Style.current.bigPadding
+
+    background: StatusDialogBackground {
+        color: Theme.palette.baseColor4
+
+        Rectangle {
+            anchors.left: parent.left
+            anchors.right: parent.right
+            anchors.top: parent.top
+            height: parent.radius
+            color: parent.color
+        }
+    }
+
+    contentItem: StackLayout {
+        id: stackLayout
+
+        // communities
+        ColumnLayout {
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                Layout.alignment: Qt.AlignCenter
+                visible: communitiesView.count == 0
+                horizontalAlignment: Text.AlignHCenter
+                verticalAlignment: Text.AlignVCenter
+                color: Theme.palette.directColor1
+                text: qsTr("%1 hasn't joined any communities yet").arg(d.mainDisplayName)
+            }
+
+            StatusGridView {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                id: communitiesView
+                cellWidth: width/2
+                cellHeight: cellWidth/2
+                visible: count
+                model: SortFilterProxyModel {
+                    sourceModel: root.isCurrentUser ? root.communitiesModel : null // TODO show other users too
+                    filters: ValueFilter {
+                        roleName: "joined"
+                        value: true
+                    }
+                    sorters: [
+                        RoleSorter {
+                            roleName: "amISectionAdmin"
+                            sortOrder: Qt.DescendingOrder // admin first
+                        },
+                        StringSorter {
+                            roleName: "name"
+                            caseSensitivity: Qt.CaseInsensitive
+                        }
+                    ]
+                }
+                ScrollBar.vertical: StatusScrollBar { }
+                delegate: StatusListItem { // TODO custom delegate
+                    width: GridView.view.cellWidth - Style.current.smallPadding
+                    height: GridView.view.cellHeight - Style.current.smallPadding
+                    title: model.name
+                    statusListItemTitle.font.pixelSize: 17
+                    statusListItemTitle.font.bold: true
+                    subTitle: model.description
+                    tertiaryTitle: qsTr("%n member(s)", "", model.members.count)
+                    asset.name: model.image ?? model.name
+                    asset.isImage: asset.name.includes("data")
```

Comment:
`asset.name.startsWith("data:image/")` might be safer

Reply:
Right, will fix

Reply:
Fixed